File: FlashCards-SPA/src/app/core/_models/learnConfiguration.ts
import {LearnTypeEnum} from './enums/learnTypeEnum';
import {FlashcardShort} from './_dtos/fromServer/flashcardShort';
import {FlashcardForLearn} from './_dtos/fromServer/flashcardForLearn';

export class LearnConfiguration {
  learnType: LearnTypeEnum;
  drawnFlashcards: FlashcardForLearn[];
  lessonName: string;
  flashcards: any[];
}

File: FlashCards.Models/DTOs/ToClient/Learn/LearnConfiguration.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FlashCards.Models.DTOs.ToClient.Learn
{
    public class LearnConfiguration
    {
        public ICollection<FlashcardForLearn> DrawnFlashcards { get; set; }

        public ICollection<object> FlashcardsForLearn { get; set; }

        public int LearnType { get; set; }

        public string LessonName { get; set; }
    }
}
